src/models/generated/Transformer3W.jl

```diff
@@ -0,0 +1,326 @@
+#=
+This file is auto-generated. Do not edit.
+=#
+
+#! format: off
+
+"""
+    mutable struct Transformer3W <: ACBranch
+        name::String
+        available::Bool
+        primary_secondary_arc::Arc
+        secondary_tertiary_arc::Arc
+        primary_tertiary_arc::Arc
+        star_bus::ACBus
+        active_power_flow_primary::Float64
+        reactive_power_flow_primary::Float64
+        active_power_flow_secondary::Float64
+        reactive_power_flow_secondary::Float64
+        active_power_flow_tertiary::Float64
+        reactive_power_flow_tertiary::Float64
+        r_primary::Float64
+        x_primary::Float64
+        r_secondary::Float64
+        x_secondary::Float64
+        r_tertiary::Float64
+        x_tertiary::Float64
+        rating::Union{Nothing, Float64}
+        r_12::Float64
```

Comment:
are these for the Y equivalent?

Reply:
Yes, the r_primary and those are the ones that are the equivalent, the r12 and those, are the direct params from PSSE